lib/scala/interpreter-dsl/src/main/java/org/enso/interpreter/dsl/BuiltinsProcessor.java

```diff
@@ -0,0 +1,211 @@
+package org.enso.interpreter.dsl;
+
+import org.apache.commons.lang3.StringUtils;
+import org.openide.util.lookup.ServiceProvider;
+
+import javax.annotation.processing.*;
+import javax.lang.model.SourceVersion;
+import javax.lang.model.element.*;
+import javax.tools.JavaFileObject;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+/**
+ * The processor used to generate code from the methods of the runtime representations annotated
+ * with {@link Builtin}.
+ *
+ * <p>It represents the first stage in generating the final builtin nodes. The processor will
+ * generate a corresponding @BuiltinMethod class which, in turn, will get processed by another
+ * processor and deliver a RootNode for the method.
+ */
+@SupportedAnnotationTypes("org.enso.interpreter.dsl.Builtin")
+@SupportedSourceVersion(SourceVersion.RELEASE_11)
```

Comment:
I suggest to override the `getSourceVersion()` method and return `SourceVersion.latest()`. Please do that also with other processors - otherwise we continue to get the nasty warnings that (our own) annotation processors don't support Frgaal's source 18.